multimedia/QMPlay2/Portfile

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+name                QMPlay2
+
+if {${os.platform} eq "darwin" && ${os.major} < 11} {
+    PortGroup       qt4 1.0
+
+    github.setup    zaps166 QMPlay2 b76e6ea46b4a32bb2becc9444ddc6f5fdff699cd
+    version         2018.12.08
+    revision        0
+    checksums       rmd160  721638bc3641199387ec5e371dd22ddabe3a1c92 \
+                    sha256  2f1f9af17e7a0dd0ce6a12eb35f425cbc43b6869cb9a32bedfef3cca70908bc6 \
+                    size    1228156
+    github.tarball_from archive
+
+    patchfiles-append \
+                    0001-macOS-unbreak-build-with-Qt4.patch \
+                    0002-MainWidget-fix-for-missing-qt_mac_set_dock_menu.patch \
+                    0003-ScreenSaver-fix-IOKit-defines.patch \
+                    0004-PlayClass-do-not-use-QRawFont.patch \
+                    0005-Revert-some-broken-code.patch \
+                    0006-FFMpeg-fix-compatibility-with-modern-FFMpeg.patch \
+                    0007-Fix-Qt-paths.patch
+
+    configure.args-append \
+                    -DDEFAULT_QT5=OFF \
+                    -DUSE_JEMALLOC=OFF \
+                    -DUSE_OPENGL2=OFF \
+                    -DUSE_TEKSTOWO=ON \
+                    -DUSE_FFMPEG_VTB=OFF
+
+    compiler.cxx_standard   2011
+
+    post-destroot {
+        system "install_name_tool -id ${applications_dir}/${name}.app/Contents/MacOS/libqmplay2.dylib \
+                    ${destroot}${applications_dir}/${name}.app/Contents/MacOS/libqmplay2.dylib"
+        system "install_name_tool -change ${prefix}/lib/libqmplay2.dylib \
+                    ${applications_dir}/${name}.app/Contents/MacOS/libqmplay2.dylib \
+                    ${destroot}${applications_dir}/${name}.app/Contents/MacOS/QMPlay2"
+        foreach dylib [ exec find ${destroot}${applications_dir}/${name}.app/Contents/MacOS/modules -name "\*.dylib" ] {
+            regsub ":$" ${dylib} "" destroot_dylib_path
+            regsub ${destroot} ${destroot_dylib_path} "" dylib_path
+            system "install_name_tool -id ${dylib_path} ${destroot_dylib_path}"
+            system "install_name_tool -change ${prefix}/lib/libqmplay2.dylib ${applications_dir}/${name}.app/Contents/MacOS/libqmplay2.dylib \
+                ${destroot_dylib_path}"
+        }
+    }
+
+    variant jemalloc description "Use Jemalloc" {
+        depends_lib-append \
+                    port:jemalloc
+        configure.args-replace \
+                    -DUSE_JEMALLOC=OFF -DUSE_JEMALLOC=ON
+    }
+} else {
+    PortGroup       qt5 1.0
+
+    qt5.depends_component qtdeclarative qttools qttranslations
+
+    github.setup    zaps166 QMPlay2 da6036373a72eab4c346b09e77207c94a35a6cd0
```

Comment:
I am starting to feel like a broken record.... released versions only please. You know how it works by now so please just follow the guidelines. The Qt5 versions for sure can be the latest release; for the Qt4 compatible version just use the latest release that supports it.

Reply:
@reneeotten There is no release for that branch. It is a specific branch for Qt4, which upstream explicitly advised to use.
Let me do ppc support in an optimal way, please, and not pick a totally random release and then add 50+ patches on top just to satisfy an arbitrary requirement :)

Reply:
For Qt5 that is exactly the commit of the latest release. For w/e reason tarball did not want to download for me. I will gladly change it from commit to a tag for Qt5, I just could not make it work.